modules/rosa-quick-start-revoke-admin-privileges.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * rosa_quick_start/rosa-quick-start.adoc
+
+[id="rosa-quick-start-revoke-admin-privileges_{context}"]
+= Revoking administrator privileges for a user
```

Comment:
we should remove the second sentence here since we have a section for revoking cluster access below.

Reply:
@0kashi can you please expand on your suggestion? Which sentence are you recommending that we remove?

Reply:
You can ignore this.  I was just being pedantic.